src/ripple/rpc/handlers/Version.h

```diff
@@ -0,0 +1,63 @@
+//------------------------------------------------------------------------------
```

Comment:
I'm not keen on having two files named `Version.h` that are one directory level away from each other.